shell/components/form/KeyValue.vue

```diff
@@ -550,9 +573,10 @@ export default {
         </div>
       </template>
       <template
-        v-for="(row,i) in rows"
+        v-for="(row,i) in filteredRows"
         v-else
       >
+        <!-- Key -->
         <div
           :key="i+'key'"
```

Comment:
We have several of these errors for each component. We should make a round to correct all of them or refactor when necessary.
For this specific case, Storybook automatically gets the args, including the slots. The error refers to the slot name `key`. It took me a while to figure it out. I am not entirely sure why we have such issue, but this would rather fall in reserved names and conventions.